Sources/Rendering/Misc/FullScreenRenderWindow/index.js

```diff
@@ -94,8 +94,14 @@ function vtkFullScreenRenderWindow(publicAPI, model) {
   model.interactor.initialize();
   model.interactor.bindEvents(model.container);
 
+  const superClass = { ...publicAPI };
+
   // Expose background
-  publicAPI.setBackground = model.renderer.setBackground;
+  publicAPI.setBackground = (background) => {
+    // Synchronize the setting of background from here to renderer
+    model.renderer.setBackground(background);
+    superClass.setBackground(background);
+  };
```

Comment:
Actually, it does not seem to make sense to store in vtkFullScreenRenderWindow the background.

Maybe you could do:
```
publicAPI.setBackground = model.renderer.setBackground
publicAPI.getBackground = model.renderer.getBackground
...
extend(...) {
Object.assign(model, defaultValues(initialValues));
...
vtkFullScreenRenderWindow(...);
// we do not want to "store" background, only forward it to renderer
initialValues.background = model.background;
delete model.background;
}
```
...